docs/observability/reliability-management-slo/create-slo.md

```diff
@@ -260,6 +219,64 @@ _view=sumologic_slo_output
 | where service="ingestion"
 ```
 
+
+## Managing your SLOs
+
+Below are some best practices for managing your SLOs. To get to your SLOs list, click **Manage Data** > **Monitoring** > **SLOs** tab.
+
+### Tags and Filters
+
+You can add key/value pair tags to your SLOs to allow you to better organize and filter them. You might find it useful to add tags for `team`, `service`, and `application`, for example.
+
+:::note Limitations
+- Tag keys cannot start with the prefixes `sumo.` or `_`
+- Tag keys can only contain letters, numbers, `_`, `.`, `/`, `+`, `-`, `@`
+- Tag values can only contain letters, white space, numbers, `_`, `.`, `/`, `=`, `+`, `-`, `@`
+- You can associate a maximum of 50 tags per SLO.
+:::
+
+#### Add a Tag
+
+To add a tag(s) to an existing SLO:
+
+1. Click on any SLO line item in your list, then click **Edit**.
+1. Scroll down to section **(3) SLO Details**. Click on **Tags (Optional)** and type in a new tag or select an existing tag.
+1. Click **Save**.
+
+To add a tag while creating a new SLO:
+
+1. Click **Add** > **New SLO**.
+1. After you've filled out sections **1** and **2**, scroll down to section **(3) SLO Details**. Click on **Tags (Optional)** and type in a new tag or select an existing tag.
+1. Click **Save**.
+
+#### Filter SLOs By Tags
+
+After you've added a tag, you'll see it populate in the **Tags** column next to your SLO in the list.
+
+1. Click **Add a filter** at the top of the screen, then click **Tag**.<br/><img src={useBaseUrl('img/observability/slo-tags.png')} alt="slo-tags.png" width="400"/>
+1. Scroll through the list of tags or type in the tag name you're looking for.<br/><img src={useBaseUrl('img/observability/slo-tags.gif')} alt="slo-tags.gif" />
+
+If you run a query with multiple values for same tag key, they are `OR`'d. Tag filters for different tag keys are `AND`'d.
+
+In this tag filter example query below, it's looking for SLOs where the `app` is either `sumo+1` OR `sumologic`, AND the `event` is `api-call`.
+
+<img src={useBaseUrl('img/observability/slo-tags-query.png')} alt="slo-tags-query.png" width="500"/>
+
+### Save filter
+
+You can create and save custom filter views, allowing you to focus on the SLOs and insights most important to you. In this example, we'll create a view that contains the filters we've created above.
```

Comment:
We can mention that max 10 saved views are allowed per user.

Reply:
Also worth mentioning that saved views are per user (i.e. are private). We don't support views at org/account level and saved views can't be shared with other users natively in the product.

Reply:
good call!